micziz/YTSP-Black
# Update installer script (#668)

* Update install.sh to be modular

* Further modularity

* Bugfix

* Bugfix pt.2

* Implement jumping to latest version and MacOS

* Bugfix maybe

* Cut down on clears

* Changed something I forgot

* Tiniest bugfix ever

* Make pip quiet!

* Fix early exits (hopefully)

* Fix early exits pt.2

* bugfix pt 80

* Fix issue with install_python_requirements

* Fix obvious issue

I have little bash experience...

* Spacing issue

* Production ready -- Fully tested on debian

* Add check for installations from .zips

* Add fail case to install_macos

* Add check for pre-installed homebrew

* Another check I forgot👍

* oops- good thing I tested it

* Add check for 'YT-Spammer-Purge' in origin url

* Exit after done installing

* Better check for git install

* Update install.sh

Co-authored-by: Daniel Ziegenberg <daniel@ziegenberg.at>

* Apply suggestions from code review

Co-authored-by: Daniel Ziegenberg <daniel@ziegenberg.at>

* Catch  errors

* Add check for python requirements

* Check if tkinter is intalled

* /usr/bin/env bash ( checked out #674 )

* Add confirmation check to replace sleeps

* Update README.md with install instructions

* Fix URL

* Adjust GIF size

* Add CLI options

* Implement more CLI options

Co-authored-by: micziz <99438936+micziz@users.noreply.github.com>
Co-authored-by: Daniel Ziegenberg <daniel@ziegenberg.at>

# Related Issue/Addition to code
Please delete options that are not relevant.

- Fixes #
- Issue/Addition to code, goes here.

## Type of change

Please delete options that are not relevant.


# Proposed Changes
- change 1
- change 2

# Additional Info
- any additional info for context

## Checklist:

- [ ] My code follows the style guidelines of this project and have read CONTRIBUTING.md

# Screenshots

Original | Updated
:----------------------:|:-----------:
** original screenshot  | ** updated screenshot **
